include/seqan3/core/simd/simd_algorithm.hpp

```diff
@@ -36,6 +39,60 @@ constexpr simd_t iota_impl(scalar_t const offset, std::integer_sequence<scalar_t
     return simd_t{static_cast<scalar_t>(offset + I)...};
 }
 
+/*!\brief Transforms a batch of ranges into simd types (Structure-of-Arrays) and writes them into the output iterator.
+ * \tparam simd_t        The simd type; must model seqan3::simd::simd_concept.
+ * \tparam output_iter_t The output iterator type; must model std::OutputIterator with `simd_t`.
```

Comment:
why not std::ranges::OutputIterator?

Reply:
Because the iterator concepts are not defined in the std::ranges namespace, are they?

Reply:
you are right